packages/scratch-gui/src/lib/libraries/extensions/index.jsx

```diff
@@ -32,7 +31,6 @@ import videoSensingIconURL from './videoSensing/video-sensing.png';
 import videoSensingInsetIconURL from './videoSensing/video-sensing-small.svg';
 
 // import posenetIconURL from './posenet/video-sensing.png';
-import poseHandInsetIconURL from './posenet/pose-hand-small-3.svg';
```

Comment:
Nice! You deleted these lines and copied over the assets!

You can then also entirely delete the `./posenet` directory reference here.

Reply:
I can't delete it all since there are some images for poseBody and poseFace in it still, but I just deleted everything there for poseHand!